src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,89 @@
+package duke.tasks;
+// import java.sql.Date;
+// import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task{
```

Comment:
According to the coding standards, we need descriptive header comments for all public classes as well as their methods. I noticed this with other public classses as well.